tests/src/ExposedServerFunctions.jsx

```diff
@@ -1,14 +1,19 @@
 import Nullstack from 'nullstack'
 
 async function api(method) {
```

Comment:
todo: add new test case to cover text/plain and application/json as per Mortaro's request